lib/l10n/intl_ro.arb

```diff
@@ -5,7 +5,6 @@
   "buttonSend": "Trimitere",
   "buttonApply": "Aplică",
   "buttonSet": "Setează",
-
```

Comment:
Don't ctrl+alt+L this file. The newlines between categories are intentional and should stay that way.

Reply:
I noticed that there were some missing spaces after : that were added below, you can keep those but add back the newlines.